### ISSUE TITLE
The program activities should display on the frontend in a calendar representation

### ISSUE THREAD

**Issue by seidior** [OpenTree-Education/rhizone-lms#428]
## Description

Once the work for issue #427 is complete, we need to transform the graphical representation of the data in the frontend into a `react-big-calendar` display.

## Details and resources

The calendar should show all activities on the calendar in every view mode at appropriate times. Optionally, there should be a drop-down to select which program the user would like to view.

## Checklist

- [x] This issue is linked to the appropriate project.
- [x] This issue has appropriate labels.

**Comment by camianderson** [comment]
Here's the current status of this ticket:

- The calendar is rendering on the calendar route in app.
- When we click in the event a pop up (window.alert) is showing up.
- In the pop up shows the title and the description of the activity.

Next step is to try to implement a modal to replace the window.alert.

**Comment by seidior** [comment]
I'm breaking the dialog feature into another issue to simplify this ticket. Hopefully this makes life easier! Go ahead and remove the code that displays a JavaScript alert, clean up the code, and implement the changes from ticket #427 once its PR has been merged, then let's open a PR for this issue.

**Comment by seidior** [comment]
Here's the latest status of the ticket:

- The PR is still in review, but changes were made to address the feedback from others.
- There are no outstanding to-do items for this ticket, but work will need to be done to address any additional issues that might pop up.
- It's up to date with `main`.
